### ISSUE TITLE
[Bug] Training files does not update when using continue_path argument

### ISSUE THREAD

**Issue by willfant** [coqui-ai/TTS#1396]
## 🐛 Description
The files best_model generated by the training when using the --continue_path are not updating. It seems that when resuming the training it gets the last checkpoint but after a couple of hours I can see that the best_model_<step>.pth.tar or best_model.pth.tar were not changed. This happens at least when we fine-tune a pre-trained model (I used glowtts). I ubuntu 20.04 but for the training it was used google colab. For the google colab, the files are stored in the google drive. No errors generated. Am I missing something in the command?

### To Reproduce

1 .Execute python train_glowtts.py  --restore_path  /content/drive/MyDrive/AI/dataset/pre_model/model_file.pth.tar (the pre trained glowtts model)
2. Stop after a few time
3. execute python train_glowtts.py --continue_path "/content/drive/MyDrive/run-March-14-2022_11+26PM-0cf3265a" resume the training 

### Expected behavior

It should resume the training and uptade the model.

### Environment

- 🐸TTS Version : 0.6.1
- Python version: 3.8
- OS: Ubuntu 20.04 / Google Colab

**Comment by willfant** [comment]
> Hey,
> 
> For the step 3. you provide nothing to the `--restore_path` option, is that a typo?
> 
> Is it possible that the loss you get by continuing the training is just higher than the loss the model got during the initial training and thus the best_model isn't updated? Does training continue to generate more recent checkpoints? (something like `checkpoint_XXXXX.pth.tar`)

Sorry my bad. My mistake on copy paste. the "--restore_path" is not in the arguments. I edited the orginial post.
For what I understood we should usse onle the --continue_path to resume the fine tune right? If yes, then it's really updating the outputs. not even recent checkpoints. The last best_model_<step> I have is the last one since I stopped the training.

**Comment by WeberJulian** [comment]
> If yes, then it's really updating the outputs. not even recent checkpoints.

What do you mean? 

Also, could you confirm that training with --continue_path generates more `checkpoint_XXXXX.pth.tar`? What's the first step number after continuing training?

**Comment by willfant** [comment]
> you share the logs of th

Here is the folder content --> https://cutt.ly/6A57ECO

`ls /content/drive/MyDrive/run-March-14-2022_11+26PM-0cf3265a`
![image](https://user-images.githubusercontent.com/63214189/158406904-3b8e6948-a955-4c18-bb4e-8172ad9909d5.png)

How can I see the logs of the `--continue_path`?

Thanks btw!

**Comment by WeberJulian** [comment]
> > you share the logs of th
> 
> Here is the folder content --> https://cutt.ly/6A57ECO
> 
> `ls /content/drive/MyDrive/run-March-14-2022_11+26PM-0cf3265a`
> ![image](https://user-images.githubusercontent.com/63214189/158406904-3b8e6948-a955-4c18-bb4e-8172ad9909d5.png)
> 
> How can I see the logs of the `--continue_path`?
> 
> Thanks btw!
> 

Thanks, I meant the training logs

**Comment by mbarnig** [comment]
I experienced the same issue with VITS and Glow-TTS. I reported about the problem in Gitter on March 10, 2020.

**Comment by willfant** [comment]
> > > you share the logs of th
> > 
> > 
> > Here is the folder content --> https://cutt.ly/6A57ECO
> > `ls /content/drive/MyDrive/run-March-14-2022_11+26PM-0cf3265a`
> > ![image](https://user-images.githubusercontent.com/63214189/158406904-3b8e6948-a955-4c18-bb4e-8172ad9909d5.png)
> > How can I see the logs of the `--continue_path`?
> > Thanks btw!
> 
> Thanks, I meant the training logs

Hey sorry, tbh I don't know were the logs are. Are that event files? Are that the file I should sens to you?

Thanks!

**Comment by WeberJulian** [comment]
Ok, don't worry about the logs, I'll try to reproduce tomorrow. I'll keep you two posted.

**Comment by mbarnig** [comment]
> > I experienced the same issue with VITS and Glow-TTS. I reported about the problem in Gitter on March 10, 2020.
> 
> It was fixed back then? Or did you find a workaround?

Sorry, my mistake. My report was 2022, not 2020.

**Comment by erogol** [comment]
Fixed in the dev and soon will be merged. I close this for now.
